client/my-sites/upgrades/checkout-thank-you/header.jsx

```diff
@@ -21,10 +21,7 @@ const CheckoutThankYouHeader = React.createClass( {
 	propTypes: {
 		isDataLoaded: React.PropTypes.bool.isRequired,
 		primaryPurchase: React.PropTypes.object,
-		selectedSite: React.PropTypes.oneOfType( [
-			React.PropTypes.object,
-			React.PropTypes.bool
-		] )
+		selectedSite: React.PropTypes.object.isRequired
```

Comment:
I think we should keep the boolean here, I tried clearing my localStorage and reloading the thank you page and had an error:
![image](https://cloud.githubusercontent.com/assets/230230/13740962/c7de389a-e9d3-11e5-892c-1af17321d60b.png)